controllers/budget_controller.py

```diff
@@ -91,31 +92,73 @@ def print_budget(budget: Budget) -> None:
-    print(f"TOTAL ACTIVITY:   {round(budget.total_activity, 2)}")
-    print(f"TOTAL AVAILABLE:  {round(budget.total_budgeted + budget.total_activity, 2)}")
+    print(f"\n{style.tYELLOW}{style.fBOLD}{style.fUNDERLINE}")
+    print(f"Here is your budget \"{budget.name}\"{style.RESET}")
+    print(f"{style.tYELLOW}{style.fUNDERLINE}MONTH: >>month<<{style.RESET}")  # TODO: fill {month} with the right data
+
+    head_table = PrettyTable()
+    head_table.add_column("TO BE BUDGETED", [
+        "",
+        f" TO BE BUDGETED: >>to_be_budg<<         ",  # TODO: fill {to_be_budg}
+        f" INFLOWS: >>inflows<<"             # TODO: fill {inflows}
+    ])
+    head_table.add_column("TOTAL BUDGETED", [
+        f"    TOTAL   ",
+        f"    BUDGETED",
+        f"%.2f" % budget.total_budgeted
+    ])
+    head_table.add_column("TOTAL ACTIVITY", [
+        f"   TOTAL   ",
+        f"   ACTIVITY",
+        f"%.2f" % budget.total_activity
+    ])
+    head_table.add_column("TOTAL AVAILABLE", [
+        f"  TOTAL    ",
+        f"  AVAILABLE",
+        f"%.2f" % (budget.total_budgeted + budget.total_activity)
+    ])
+    # NOTE: Above '%.2f' is made for printing two decimal places including zeros at the end.
+    # (When we use "round(x, 2) then it prints for example 7.0 instead of 7.00)
+
+    head_table.align = "r"
+    head_table.align["TO BE BUDGETED"] = "l"
+    head_table.header = False
+    head_table.border = False
+
+    print(f"{style.tYELLOW}", end="")   # end="" causes that there is no empty newline after this line of code.
+    print(head_table, end="")
+    print(f"{style.RESET}")
+
 
     # Print the table
     print(table_budget)
 
 
 def print_budget_bar_chart(budget: Budget) -> None:
     bar_chart = PrettyTable()
-    bar_chart.field_names = [" id, CATEGORY", "ACTIVITY", "BAR CHART"]
+    bar_chart.field_names = ["id, CATEGORY", "ACTIVITY", "BAR CHART"]
     bar_chart.align = "l"
     bar_chart.align["ACTIVITY"] = "r"
     bar_chart.float_format = "1.2"
 
+    i_color = 31
     for parent in budget.parent_categories:
         for category in parent.categories:
             activity_amount = sum(activity.amount_outflow for activity in category.transactions)
-            bar = int(round(activity_amount / 100)) * "#"  # Each 100 PLN is a single "#"
+            if activity_amount % 100 > 75:  # % gives the rest of the division
+                bar_ending = "█"
+            elif activity_amount % 100 > 50:
+                bar_ending = "▊"                # also could be "▓"
+            elif activity_amount % 100 > 25:
+                bar_ending = "▌"                # also could be "▒"
+            elif activity_amount % 100 > 0:
+                bar_ending = "▎"                # also could be "░"
+            elif activity_amount % 100 == 0:
+                bar_ending = ""
+            bar = f"\033[{i_color}m{int(activity_amount//100)*'█'}{bar_ending}{style.RESET}"
```

Comment:
Moim zdaniem to się powinno proporcjonalnie wyliczać, a nie tak jak obecnie - floor division.

Lepiej byłoby, gdyby kategoria z największymi wydatkami to byłoby 100 "barów". Pozostałe proporcjonalnie mniej.